src/styles/utils/_variables.scss

```
@@ -0,0 +1,15 @@
+$boxes-color:#eee;
+$boxes-width: 100px;
+$boxes-in-row: 7;
+$padding-size: 10px;
+$gap-size: 1px;
+$block-width: $boxes-width * $boxes-in-row + $gap-size * 7 + $padding-size * 2;
+$days: mon, tue, wed, thu, fri, sat, sun;
+$week:
+  "mon" 0,
+  "tue" $boxes-width + $gap-size,
+  "wed" ($boxes-width + $gap-size) * 2,
+  "thu" ($boxes-width + $gap-size) * 3,
+  "fri" ($boxes-width + $gap-size) * 4,
+  "sat" ($boxes-width + $gap-size) * 5,
+  "sun" ($boxes-width + $gap-size) * 6;
```

```suggestion
  "tue" 1,
  "wed: 2,
  "thu": 3,
  "fri" : 4,
  "sat": 5,
  "sun": 6;
```
Move calculations to the loop